src/service/controllers/ucs.js

```diff
@@ -112,18 +112,19 @@ searchContacts = (req, res) => {
 	if (searchedTerm.indexOf('\\') === 0) {
 		searchedTerm = searchedTerm.substring(1);
 	}
+	searchedTerm = new RegExp(searchedTerm, 'i');
```

Comment:
I guess this update can generate issues.
For example, if someone was looking for a string with some special characters from a regexp (like . or ? or ...), the behavior will be updated.
Why not simply do
searchedTerm = searchedTerm.toLowerCase();

And, then compare with with lowercase version of emailaddress, firstname or lastname (or companyname)

Reply:
Agree with Stéphane, no regex should be used for this case.
Proposal from Stéphane H sounds the good one.

Reply:
Agreed with you Stephane H. Please refer to the new Pull Request